vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -159,9 +159,8 @@ vic_cesm_put_data()
 
         // evaporation, VIC: mm, CESM: kg m-2 s-1
         // TO-DO should we incorporate bare soil evap?
```

Comment:
Is this TODO still relevant? My assumption is that WRF wants total evap, so if soil evap is already included in `OUT_EVAP` then we should be OK. If not, then we should add it.